### ISSUE TITLE
typo in readme

### ISSUE THREAD

**Issue by tbbooher** [jordanh/neurio-python#10]
You have:

```
import my_keys

# Setup authentication:
tp = neurio.TokenProvider(key=example_keys.key, secret=example_keys.secret)
```
I think you mean: **my_keys**.key
```
import my_keys

# Setup authentication:
tp = neurio.TokenProvider(key=my_keys.key, secret=my_keys.secret)
```

**Comment by jordanh** [comment]
Great catch! Thank you!

**Comment by jordanh** [comment]
Fixed by 9e03de3.
